### PR TITLE
Adding a brain damage line hits your PR balance by -10

### PR DESCRIPTION
**Pull request by Cyberboss**

### CODE REVIEW

File: tools/WebhookProcessor/github_webhook_processor.php [hunk 1]
```diff
@@ -373,6 +373,7 @@ function get_pr_code_friendliness($payload, $oldbalance = null){
 		'Feature' => -1,
 		'Balance/Rebalance' => -1,
 		'Tweak' => -1,
+		'Brain Damage Line' => -10,
```
Comment (Iamgoofball) [inline tools/WebhookProcessor/github_webhook_processor.php]: what does this even do

Comment (Cyberboss) [reply tools/WebhookProcessor/github_webhook_processor.php]: #29897

Comment (PraiseRatvar) [reply tools/WebhookProcessor/github_webhook_processor.php]: https://github.com/tgstation/tgstation/labels/Brain%20Damage%20Line

**Comment by ExcessiveUseOfCobblestone** [comment]
bye goof!

**Comment by Iamgoofball** [comment]
i cant die

**Comment by optimumtact** [comment]
no, don't ruin a good tool with memes.

**Pull request closed without merging**